File: app/src/main/java/mx/yellowme/app/simple/UmbrellaActivity.java
package mx.yellowme.app.simple;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.widget.FrameLayout;

import mx.yellowme.app.R;

/**
 * Created by migdonio on 8/2/17.
 */

public class UmbrellaActivity extends AppCompatActivity {

    private UmbrellaFragment umbrellaFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_example1);

        if(findViewById(R.id.fragment_container) != null) {
            if(savedInstanceState != null) {
                return;
            }

            umbrellaFragment = new UmbrellaFragment();
            umbrellaFragment.setArguments(getIntent().getExtras());
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fragment_container, umbrellaFragment).commit();
        }
    }
}

File: app/src/main/java/mx/yellowme/app/simple/UmbrellaActivity.kt
package mx.yellowme.app.simple

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View

import mx.yellowme.app.R

class UmbrellaActivity : AppCompatActivity() {

    private var umbrellaFragment: UmbrellaFragment? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_example2)

        if (findViewById<View>(R.id.fragment_container) != null) {
            if (savedInstanceState != null) {
                return
            }
            umbrellaFragment = UmbrellaFragment()
            umbrellaFragment?.arguments = intent.extras
            supportFragmentManager.beginTransaction()
                    .add(R.id.fragment_container, umbrellaFragment).commit()
        }
    }
}
